static/js/reminder.js

```diff
@@ -96,7 +99,7 @@ export function schedule_message(request = compose.create_message_object()) {
     };
     const error = function (response) {
         $("#compose-textarea").prop("disabled", false);
-        compose_error.show(_.escape(response), $("#compose-textarea"));
+        compose_error.show_error_message(response, $("#compose-textarea"));
```

Comment:
This seems like a bug; the second parameter is supposed to be a string...

Reply:
thanks!